onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.py

```diff
@@ -0,0 +1,53 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import ctypes
+import os
+import sys
+
+build_dir = os.environ.get("KERNEL_EXPLORER_BUILD_DIR", None)
+if build_dir is None:
+    raise ValueError("Environment variable KERNEL_EXPLORER_BUILD_DIR is required")
+
+if not os.path.exists(build_dir):
+    raise ValueError(f"KERNEL_EXPLORER_BUILD_DIR ({build_dir}) points to nonexistent path")
+
+build_dir = os.path.realpath(build_dir)
+search_paths = [build_dir]
+
+# kernel explorer hooks up some underlaying functions for microbenchmark purpose. Some utility functions might also be
```

Comment:
Maybe something like this:

As Kernel Explorer makes use of utility functions in ONNXRuntime, we dlopen all relevant libraries to bring all required symbols into global namespace, so that we don't need to worry about linking.